sidemenu-addon/src/main/java/org/vaadin/teemusa/sidemenu/SideMenu.java

```diff
@@ -243,6 +246,18 @@ public void setMenuCaption(String caption) {
 	}
 
 	/**
+	 * Sets the title text for the menu
+	 * 
+	 * @param caption
+	 *            menu title
+	 */
+	public void setMenuImage(String caption, Resource source) {
```

Comment:
Is the menu image something you can have at the same time as menu caption? If not, should these two functionalities merged somehow together?